src/term/flavour_py/mod.rs

```diff
@@ -0,0 +1,98 @@
+mod order_enums;
+pub mod to_lang;
+
+use indexmap::IndexMap;
+use interner::global::GlobalString;
+
+use crate::term::Name;
+
+use super::Op;
+
+#[derive(Clone, Debug)]
+pub enum Term {
+  // "None"
+  None,
+  // [a-zA-Z_]+
+  Var { nam: Name },
+  // [0-9_]+
+  Num { val: u32 },
+  // {fun}(args,)
+  Call { fun: Box<Term>, args: Vec<Term> },
+  // "fun" {pat} -> {bod}
+  Lam { pat: AssignPattern, bod: Stmt },
+  // {nam} "{" {fields} "}"
+  Enum { nam: Name, fields: Vec<(Name, Term)> },
```

Comment:
If we allow keyword arguments in functions, we could have the same syntax for function calls and enums (in the end they're the same thing right?).
That's how python does it, you create an object by calling the constructor function (which is rather complicated, but for us we can just pretend it's a normal function call).